cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -155,97 +217,73 @@ func newVolume(name, path string) v1.Volume {
 	}
 }
 
-func newVolumeMount(name, path string) v1.VolumeMount {
+// newRWVolumeMount creates a read-write v1.VolumeMount to the specified location
+func newRWVolumeMount(name, path string) v1.VolumeMount {
 	return v1.VolumeMount{
 		Name:      name,
 		MountPath: path,
+		ReadOnly:  false,
 	}
 }
 
-// etcdVolume exposes a path on the host in order to guarantee data survival during reboot.
-func etcdVolume(cfg *kubeadmapi.MasterConfiguration) v1.Volume {
-	return v1.Volume{
-		Name: "etcd",
-		VolumeSource: v1.VolumeSource{
-			HostPath: &v1.HostPathVolumeSource{Path: cfg.Etcd.DataDir},
-		},
-	}
-}
-
-func etcdVolumeMount(dataDir string) v1.VolumeMount {
+// newROVolumeMount creates a read-only v1.VolumeMount to the specified location
+func newROVolumeMount(name, path string) v1.VolumeMount {
```

Comment:
Yeah, I know, but I felt it would be easier to read RO/RW in func name vs passing a bool. Such a small duplication so I thought it was worth it.